modules/grafana/Makefile

```diff
@@ -19,10 +19,25 @@ grafana/develop: grafana/cleanup grafana/setup-local-grafana-mintel grafana/setu
 # Start up a local grafana instance with no datasources configured, and sync to dashboards on your local disk
 grafana/develop-oss: grafana/cleanup grafana/setup-local-grafana-oss grafana/setup-grafana-syncer
 
+.PHONY: grafana/cleanup
+# Kill any docker containers associated with grafana/develop, and remove anything added to /tmp
+grafana/cleanup:
+	@echo "Killing ${GRAFANA_LOCAL_DOCKER_NAME}..."
+	@docker kill ${GRAFANA_LOCAL_DOCKER_NAME} || true
+	@echo "Killing ${GRAFANA_SYNC_DOCKER_NAME}..."
+	@docker kill ${GRAFANA_SYNC_DOCKER_NAME} || true
+	@echo "Removing ${TMP_GITLAB_REPO_DIRECTORY}..."
+	@rm -rf ${TMP_GITLAB_REPO_DIRECTORY}
+	@echo "Cleanup successful."
+
+grafana/aws-profile-check:
```

Comment:
This should probably have a `.PHONY: grafana/aws-profile-check` above it.

Reply:
added .PHONY to all targets in new revision (I thought it meant that the target could only be called from outside the Makefile if tagged with .PHONY, but it seems that it means the target is not a file name, which is true of all targets here)